captionsProcessor.py

```diff
@@ -0,0 +1,25 @@
+from config import Config
+from transcriptLoader import getCombinedTranscripts
+from topicExtractor import retrieveTopics
+from utils import printAndLog
+
+
+def main():
+    config = Config()
+    config.setFromEnv()
+
+    videoToUse = "New Google Assignments in Canvas"
+
+    printAndLog(f"Retrieving Transcript for {videoToUse}")
+
+    transcriptToUse = getCombinedTranscripts(
+        videoToUse,
+        captionsFolder=config.captionsFolder,
+    )
+
+    printAndLog(f"Retrieving Topics for {videoToUse}")
+
+    topicsOverTime, topicModel = retrieveTopics(videoToUse, transcriptToUse, config=config)
```

Comment:
Before passing the `transcriptToUse` response to `retrieveTopics` make some basic validation check. what if `transcriptToUse` returns `None`.  This is best practice and should account for error case as well.